src/main/fc/rc_controls.c

```diff
@@ -83,11 +83,12 @@ PG_RESET_TEMPLATE(rcControlsConfig_t, rcControlsConfig,
     .yaw_control_reversed = false,
 );
 
-PG_REGISTER_WITH_RESET_TEMPLATE(armingConfig_t, armingConfig, PG_ARMING_CONFIG, 1);
+PG_REGISTER_WITH_RESET_TEMPLATE(armingConfig_t, armingConfig, PG_ARMING_CONFIG, 2);
```

Comment:
Incrementing the PG version is not needed if a new parameter is added that gets set to 0, afaik

Reply:
Per comment from @ledvinap we always update PG when the structure is updated.